cli/crates/cli/src/cli_input.rs

```diff
@@ -283,6 +283,38 @@ impl StartCommand {
     }
 }
 
+#[derive(Debug, clap::Args)]
+pub struct IntrospectCommand {
+    /// GraphQL URL to introspect. Defaults to "http://localhost:4000"
```

Comment:
url should be required without a default value IMO

Reply:
This is quite useful if you're in a local context. We default to `http://localhost:4000`, and you just want to display the current schema from the running service.

Reply:
So you have a local subgraph running, you want to display and output the schema to somewhere, all you need to do is:

```console
grafbase subgraph introspect
```

Reply:
Agreed. Will remove that.